CMakeLists.txt

```diff
@@ -94,6 +94,8 @@ option ( enable-dsound "compile DirectSound support (if it is available)" on )
 option ( enable-wasapi "compile Windows WASAPI support (if it is available)" on )
 option ( enable-waveout "compile Windows WaveOut support (if it is available)" on )
 option ( enable-winmidi "compile Windows MIDI support (if it is available)" on )
+option ( enable-haiku "compile Haiku MediaKit audio support (if it is available)" on )
+option ( enable-midikit2 "compile Haiku MidiKit2 MIDI support (if it is available)" on )
```

Comment:
Both of these cmake options should be wraped inside a `if ( CMAKE_SYSTEM MATCHES "Haiku" )`, as we do it below, since they are platform-specific options.